_integration-test/run.sh

```diff
@@ -133,7 +133,7 @@ $dcr --no-deps web python3 /etc/sentry/test-custom-ca-roots.py
 source ./custom-ca-roots/teardown.sh
 echo "${_endgroup}"
 
-echo "${_group}Ensure customizations work"
-$dcr web bash -c "if [ ! -e /created-by-enhance-image ]; then exit 1; fi"
-$dcr web python -c "import ldap"
+echo "${_group}Ensure customizations are not present yet"
+! $dcr web bash -c "if [ ! -e /created-by-enhance-image ]; then exit 1; fi"
```

Comment:
Please avoid negative negatives :)
It does not help my eyes, but let's hear from others too before changing.

Reply:
I generally agree, but decided that just adding the `!` in front would be better for maintenance. If the condition were to change we can just replace it in both files without thinking about how it can be negated.

```suggestion
$dcr web bash -c "if [ -e /created-by-enhance-image ]; then exit 1; fi"
```

Reply:
What other conditional?

Reply:
I clarified the symmetry between the positive and negative version of those test statements and added a hint to ensure it is maintained in the future.